### PR TITLE
Gst-vaapi support av1 decode 8bit

### PR DESCRIPTION
**Pull request by FocusLuo**
Gst-vaapi support av1 decode 8bit

### CODE REVIEW

**Comment by FocusLuo** [comment]
depend two gst patches merged as below:
gst-vaapi:
https://gitlab.freedesktop.org/gstreamer/gstreamer-vaapi/-/merge_requests/191
gst-plugins-bad:
https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/1464

File: test/gst-vaapi/decode/av1.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+###
+### Copyright (C) 2018-2020 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
```
Comment (uartie) [inline test/gst-vaapi/decode/av1.py:8]: Since #238, please use `from ....lib.gstreamer.vaapi.util import *`

File: test/gst-vaapi/decode/av1.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from .decoder import DecoderTest
```
Comment (uartie) [inline test/gst-vaapi/decode/av1.py:9]: Since #238, please use `from ....lib.gstreamer.vaapi.decoder import DecoderTest`

**Review by uartie — CHANGES_REQUESTED** [review]
Please rebase since #238

File: test/gst-vaapi/util.py [hunk 3]
```diff
@@ -101,6 +101,9 @@ def mapprofile(codec, profile):
       "main10"                : "main-10",
       "main444-10"            : "main-444-10",
     },
+    "av1-8"   : {
+      "main"                  : "main",
+    },
```
Comment (uartie) [inline test/gst-vaapi/util.py:106]: Since #238, this file has moved to lib/gstreamer/vaapi/util.py

**Comment by FocusLuo** [comment]
Artie, with gst-vaapi merged several days ago, can you help to change the status of this patch?

**Review by uartie — APPROVED** [review]

**Pull request merged**